src/main/java/camera/CameraChestDetector.java

```diff
@@ -34,16 +35,17 @@
      * @param subtraction the subtraction of this frame.
      * @return true if chest is detected, false otherwise.
      */
-    public boolean checkForChests(final Mat newFrame, final int noOfChests, final Mat subtraction) {
+    public List<Mat> checkForChests(final Mat newFrame, final int noOfChests,
+                                    final Mat subtraction) {
         Mat dest = getChestsFromFrame(bgrToHsv(newFrame));
         Mat subtracted = new Mat();
-        boolean isDetected = false;
+        List<Mat> mats = new ArrayList<>();
```

Comment:
Is this consistent with the other places where ArrayLists are initialized

Reply:
Wat bedoel je

Reply:
Link doe je generic List, rechts het actual type, dus Arraylist

Reply:
Laat maar

Reply:
I hope so